tests/services-aerospace.nix

```diff
@@ -31,6 +31,6 @@ in
       ${config.out}/user/Library/LaunchAgents/org.nixos.aerospace.plist`
 
```

Comment:
Can you update the settings above and use the new option you added and set `on-window-detected` maybe with the example value?